index.js

```diff
@@ -150,13 +150,25 @@ module.exports = function(input, map) {
     loaderUtils.getOptions(this)
   )
 
+  var userEslintPath = userOptions.eslintPath
+  var formatter = require("eslint/lib/formatters/stylish")
```

Comment:
Is this `require("eslint/lib/formatters/stylish")` redundant given the one in the try-catch below?